### DISCUSSION TITLE
It can't work on STM32G0B1CBT6 with FreeRTOS. Please help.

### DISCUSSION THREAD

**Discussion by favoritewky** [hathach/tinyusb#2704] [Q&A] [answered]
### Operating System

Windows 11

### Board

STM32G0B1CBT6

### Firmware

I make a simple project here https://github.com/hathach/tinyusb/commit/b3ee927e58cf7cd44f05db9dec41bc241ef94902

### What happened ?

I make a CDC Usb device but not work, I need help.
It will get 
```
error: redundant redeclaration of 'SystemCoreClock' [-Werror=redundant-decls]
   53 |   extern uint32_t SystemCoreClock;
```
So, I comment this line https://github.com/hathach/tinyusb/commit/b3ee927e58cf7cd44f05db9dec41bc241ef94902#r144079217

### How to reproduce ?

Download the firmware but nothing happend.

### Debug Log as txt file (LOG/CFG_TUSB_DEBUG=2)

```
.USBD init on controller 0, Highspeed = 0
sizeof(usbd_device_t) = 53
sizeof(dcd_event_t) = 12
sizeof(tu_fifo_t) = 20
sizeof(tu_edpt_stream_t) = 112
CDC init
```

### I have checked existing issues, dicussion and documentation

- [X] I confirm I have checked existing issues, dicussion and documentation.

**Comment by favoritewky** [comment]
@Commotari Please help me.

**Reply by Commotari** [reply]
the STM32G0B1 has not been checked in /merged in the official tinyusb stack so far.
you can contact me: funkmaster.whylee@gmail.com

can someone do a merge?
https://we.tl/t-q2Wnp27GJN

**Reply by HiFiPhile** [reply]
STM32G0B1 has been supported long ago, you can do a keyword search.

**Comment by HiFiPhile** [comment]
Please don't use issue for asking for help.

**Comment by favoritewky** [comment, marked as answer]
Thanks for help! It works when I change 
`#define configTIMER_TASK_PRIORITY                ( 30 )`
